test/es_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+require 'elasticsearch/model'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.